packages/pages/src/vite-plugin/modules/plugin.ts

```diff
@@ -75,10 +88,10 @@ export const buildModules = async (
   for (const [moduleName, fileInfo] of Object.entries(filepaths)) {
     logger.info = (msg, options) => {
       if (msg.includes("building for production")) {
-        loggerInfo(pc.green(`\nBuilding ${moduleName} module...`));
+        logger.info(pc.green(`\nBuilding ${moduleName} module...`));
```

Comment:
Why change this from the loggerInfo helper?

Reply:
there were two variables created loggerInfo and loggerWarn. LoggerWarn was not needed outside of the createModuleLogger() function so to clean up a little I also removed loggerInfo which was only used in these two lines.

Reply:
Ah ok. I thought this was something from logError.ts.